db/migrate/20210419142255_create_privacy_policy_acceptances.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class CreatePrivacyPolicyAcceptances < ActiveRecord::Migration[6.1]
+  class PrivacyPolicy < ApplicationRecord
+  end
+
+  class User < ApplicationRecord
+  end
+
```

Comment:
Why can't we just require these classes in here as we have already defined them in the models?

Reply:
@amazimbe Migrations should always be independent from the main application. Honestly, we should not even depend on the ApplicationRecord (but rubocop requires it...). The reason is that application code changes all the time and those changes should not affect migration. If at any point we rename User (I know, it is unlikely) this would stop new developers from running migrations.